<launch>

 <group ns="turtlebot1">
  <node name="goal_publish_1" pkg="multi_robot_action_move" type="robots_pos_publish.py" output="screen" args="turtlebot1"/>
 </group>

  <node pkg='turtlesim' name='turtlesim_node' type='turtlesim_node'>
    <remap from='turtle1/pose' to='/turtlebot1/pose' />
    <remap from='turtle1/cmd_vel' to='/turtlebot1/cmd_vel' />
    <remap from='turtle2/pose' to='/turtlebot2/pose' />
    <remap from='turtle2/cmd_vel' to='/turtlebot2/cmd_vel' />
  </node>

   <node pkg='turtlesim_nav' name='pose_to_tf' type='pose_to_tf_1.py' />

 <group ns="turtlebot2">
  <node name="goal_publish_2" pkg="multi_robot_action_move" type="robots_pos_publish.py" output="screen" args="turtlebot2"/>
 </group>

</launch>
